Source/AdUnits/Native/NativeAsset.swift

```diff
@@ -123,29 +123,20 @@ public class NativeAsset: NSObject {
     }
 }
 
-@objc public enum ImageAsset: Int {
-    case Icon = 1
-    case Main = 3
-    case Custom
+public class ImageAsset: SingleContainerInt {
```

Comment:
why do you extend SingleContainerInt ?

Reply:
I had replaced all enums with class objects as it was not compatible with ObjectiveC apps. hence took the SingleContainerInt out of signals so that it can be used by all asset objects